src/components/LoginRegister.js

```diff
@@ -0,0 +1,234 @@
+import {
+  VStack,
+  Heading,
+  Button,
+  Box,
+  FormControl,
+  InputGroup,
+  InputLeftElement,
+  InputRightElement,
+  Input,
+  FormHelperText,
+  FormLabel,
+} from '@chakra-ui/react';
+import { useState, useRef } from 'react';
+
+const LoginRegister = (props) => {
```

Comment:
Code style comment - always destructure props object, makes it easier

```suggestion
const LoginRegister = ({ activeOption, setIsLoggedIn }) => {
```